lib/tasks/server/livereload-server.js

```diff
@@ -5,6 +5,10 @@ var path        = require('path');
 var Task        = require('../../models/task');
 var SilentError = require('../../errors/silent');
 
+// default reload css extensions
+var styleExtensions = ['css', 'scss', 'sass', 'less', 'styl'];
```

Comment:
I just checked the source for the most popular css precompilers and they are not consistent in terms of registering their file extensions.

https://github.com/aexmachina/ember-cli-sass/blob/master/index.js#L21
https://github.com/gdub22/ember-cli-less/blob/master/index.js#L12
https://github.com/drewcovi/ember-cli-stylus/blob/master/index.js#L6

I tried using the same technique used in the Addon model but couldn't get to the extensions. Also search through the registry instance and did not find the extensions or any other info I could use to extract the file extensions.

Any suggestions on how to proceed?